meta-iotqa/lib/oeqa/runtime/multimedia/opencl/opencl.py

```diff
@@ -0,0 +1,10 @@
+from oeqa.oetest import oeRuntimeTest
+
+class OpenCLTest(oeRuntimeTest):
+    def test_opencl_has_library_file(self):
+        (status, output) = self.target.run('ls /usr/lib/libOpenCL*')
```

Comment:
Maybe `test -f filename` would be a cleaner way to check for the existence of a file? `ls` promises to return non-zero if an error occurred, but I think it might not be defined in the spec that a missing file counts as an error when using wildcards.